fract4d/c/model/STFractWorker.cpp

```diff
@@ -762,13 +752,13 @@ inline void STFractWorker::rectangle_with_iter(
             m_im->setIter(j, i, iter);
             m_im->setFate(j, i, 0, fate);
             m_im->setIndex(j, i, 0, index);
-            m_stats.s[PIXELS]++;
-            m_stats.s[PIXELS_SKIPPED]++;
+            ++m_stats.s[PIXELS];
+            ++m_stats.s[PIXELS_SKIPPED];
         }
     }
 }
 
-// @TODO: this is used for RENDER_THREE_D mode and seems to be unfinished or at leats worth to review
+// @TODO: this is used for RENDER_THREE_D mode and seems to be unfinished or at least worth to review
```

Comment:
yeah, never finished RENDER_THREE_D

Reply:
That might be another enhancement to add to [the list](https://github.com/fract4d/gnofract4d/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
I'm also very curious about the optimizations I wrapped with EXPERIMENTAL_OPTIMIZATIONS macro in this PR.